osu.Game/Online/Chat/MessageFormatter.cs

```diff
@@ -279,7 +279,8 @@ public enum LinkAction
         JoinMultiplayerMatch,
         Spectate,
         OpenUserProfile,
-        Custom
+        Custom,
+        OpenDirectWithSearch
```

Comment:
should be above `Custom` to organize it better. ("otherwise / else / custom" enum should be last)